facebook_connector.py

```diff
@@ -43,34 +44,41 @@ def initialize(dump_directory="."):
         _messages_file = None
 
 
-def resolve_user_id(user_id):
+def resolve_user_id(fb_provided_identifier):
     """
     Tries to map the identifier facebook provides to the name of the user
 
     Args:
-        user_id: identifier string that is provided in the messages file
+        fb_provided_identifier: identifier string that is provided in the messages file
+            This might be the user's name or his Facebook ID
```

Comment:
Perhaps offer an example of what may be passed, and indicate that something like 123456789@facebook.com may get mapped, whereas something like "Mike Dezube" would be returned unaffected.